game_data_cleaner/main.py

```diff
@@ -122,27 +130,39 @@ def _save_dfs_to_disk_or_s3(self):
                 table = pd.DataFrame.from_dict(list_of_entries)
             else:
                 table = pd.concat(list_of_entries, ignore_index=True)
+                self._make_json_game_lookup_file(table)
 
             print(f"Deleting {table_name} from memory")
             del list_of_entries
 
             print(f"Saving {table_name} to disk and uploading to S3")
 
-            table.to_pickle(
-                f"data/game_dfs_dirty/{table_name}.pkl"
-            )
+            LocalFileHandler().save_file(file_path=f"data/games/game_dfs_dirty/{table_name}.pkl", data=table)
             if ENV == "prod":
-                wr.s3.upload(
-                    f"{table_name}.pkl",
-                    f"s3://{S3_SCRAPER_BUCKET}/game_dfs_dirty/{table_name}.pkl",
-                )
+                S3FileHandler().save_file(file_path=f"games/game_dfs_dirty/{table_name}.pkl", data=table)
```

Comment:
Config opportunity.  Also seems to use the old directory structure (no `/data/`)